beacon-chain/core/altair/upgrade_test.go

```diff
@@ -73,8 +72,6 @@ func TestUpgradeToAltair(t *testing.T) {
 	preForkState := st.Copy()
 	aState, err := altair.UpgradeToAltair(context.Background(), st)
 	require.NoError(t, err)
-	_, ok := aState.(state.BeaconStateAltair)
-	require.Equal(t, true, ok)
```

Comment:
Can you explain these change? Why was this removed?

Reply:
I might be wrong, but isn't this an unnecessary type assertion?

Reply:
Not really, it is an important type assertion. We want to make sure that we are getting back the correct state type here after the upgrade method.